smart-web-server/src/main/java/org/smartdata/server/security/RangerAuthorizationManager.java

```diff
@@ -17,31 +17,92 @@
  */
 package org.smartdata.server.security;
 
+import com.google.common.collect.ImmutableMap;
 import lombok.RequiredArgsConstructor;
+import org.smartdata.ranger.SsmResourceAccessType;
 import org.smartdata.ranger.authorizer.RangerSsmAuthorizer;
 import org.smartdata.ranger.authorizer.request.RangerAuthorizeRequest;
+import org.smartdata.ranger.authorizer.request.RangerOperationDto;
 import org.springframework.security.authorization.AuthorizationDecision;
 import org.springframework.security.authorization.AuthorizationManager;
 import org.springframework.security.core.Authentication;
 import org.springframework.security.web.access.intercept.RequestAuthorizationContext;
 
-import java.util.Collections;
+import javax.ws.rs.HttpMethod;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
 import java.util.function.Supplier;
 
+import static org.smartdata.ranger.SsmRangerResourceEnum.ACTION;
+import static org.smartdata.ranger.SsmRangerResourceEnum.AUDIT;
+import static org.smartdata.ranger.SsmRangerResourceEnum.CLUSTER;
+import static org.smartdata.ranger.SsmRangerResourceEnum.RULE;
+
 @RequiredArgsConstructor
 public class RangerAuthorizationManager
     implements AuthorizationManager<RequestAuthorizationContext> {
 
+  private static final
+  Map<String, Map<String, Function<RequestAuthorizationContext, RangerOperationDto>>>
+      OPERATION_MAP =
+      new HashMap<String, Map<String,
+          Function<RequestAuthorizationContext, RangerOperationDto>>>() {{
+        //cluster
+        put("/api/v2/cluster/nodes", ImmutableMap.of(HttpMethod.GET,
+            request -> CLUSTER.getRangerOperationDto(SsmResourceAccessType.VIEW.name(), "*",
+                ImmutableMap.of())));
+        //Rules
+        put("/api/v2/rules", ImmutableMap.of(HttpMethod.POST,
```

Comment:
Consider adding mapping for the `VIEW` access type for the rules resource

Reply:
in some reason the last changes gone missing, please review this class again